doc/api/deprecations.md

```diff
@@ -660,6 +660,15 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: path._makeLong()
+
+Type: Runtime
```

Comment:
I don't know if this should be runtime- or docs- deprecated, but I think the deprecation should be in an other commit so we can cherry-pick the addition to v8.x